### PR TITLE
Color Scheme: Claudinho

### PR DESCRIPTION
**Pull request by gkal19**
- Link to your code repository: [Claudinho](https://github.com/gkal19/claudinho)
 - Link to the tags page with at least one [semver](http://semver.org) tag: [v2.0.0](https://github.com/gkal19/claudinho/releases/tag/v2.0.0)

### CODE REVIEW

**Comment by FichteFoll** [comment]
https://packagecontrol.io/docs/submitting_a_package#Step_4

**Comment by gkal19** [comment]
@FichteFoll Here are my new commits

File: channel.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
 		"https://raw.githubusercontent.com/francodacosta/sublime-php-getters-setters/master/packages.json",
 		"https://raw.githubusercontent.com/freewizard/sublime_packages/master/package_control.json",
 		"https://raw.githubusercontent.com/gcollazo/sublime_packages/master/packages.json",
+		"https://raw.githubusercontent.com/gkal19/claudinho/master/packages.json",
```
Comment (FichteFoll) [inline channel.json]: Please remove this line.

File: repository/c.json [hunk 2]
```diff
@@ -1155,6 +1155,18 @@
 			]
 		},
 		{
+			"name": "Claudinho",
+			"details": "https://github.com/gkal19/claudinho",
+			"labels": ["color scheme"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"platforms": ["windows"],
```
Comment (FichteFoll) [inline repository/c.json]: Color schemes aren't platform-dependant. You should remove this restriction.

**Comment by gkal19** [comment]
@FichteFoll the files have changed as you requested.
The code is working.

**Pull request merged**